README.md

```diff
@@ -20,15 +20,42 @@ Get started developing applications quickly with best practices using Serverless
 ## Getting started
 To get started, run the following commands:
 
-`git clone https://github.com/wizeline/serverless-fullstack`
-`cd serverless-fullstack`
-`cp example.setup.config.json setup.config.json`
+```
+git clone https://github.com/wizeline/serverless-fullstack
+cd serverless-fullstack
+touch setup.config.json
+```
+
+Adjust setup.config.json to set your application name and your aws credentials. It should contain the following structure:
+
+```
+{
+  "applicationName": "",
+  "shortApplicationName": "",
+  "awsProfiles": {
+    "dev": {
+      "accessKey": "",
+      "secretAccessKey": ""
+    },
+    "staging": {
+      "accessKey": "",
+      "secretAccessKey": ""
+    },
+    "prod": {
+      "accessKey": "",
+      "secretAccessKey": ""
+    }
+  }
+}
```

Comment:
Create a placeholder `setup.config.json` file with this content so the user doesn't need to touch-copy-paste